<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CamDesign</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>cam.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Figure" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>740</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>740</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="platemovecam">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="InputTable" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>269</x>
        <y>6</y>
        <width>441</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="PMCokButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>rotion direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>basic radius</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>offset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="PMCoffsetComboBox">
           <item>
            <property name="text">
             <string>right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>middle</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>follower shape</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>110</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="PMCRotionDirectionComboBox">
         <item>
          <property name="text">
           <string>anticlockwise</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>clockwise</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PMCbasicRadiusEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="PMCoffsetEdit"/>
       </item>
       <item>
        <widget class="QComboBox" name="PMCfollowerShapeComboBox">
         <item>
          <property name="text">
           <string>needle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>roller</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>plate</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>114</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QComboBox" name="PMCfollowerLayoutComboBox">
         <item>
          <property name="text">
           <string>outside</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>inside</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>embedded</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PMCfollowerRadiusEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>114</y>
        <width>121</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="PMCfollowerLayoutLabel">
         <property name="text">
          <string>follower layout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="PMCfollowerRadiusLabel">
         <property name="text">
          <string>follower radius</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="drawButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InputTable</class>
   <extends>QTableWidget</extends>
   <header>inputtable.h</header>
  </customwidget>
  <customwidget>
   <class>Figure</class>
   <extends>QWidget</extends>
   <header location="global">figure.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>PMCfollowerShapeComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>showFollowerRadius(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PMCokButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inputOkButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PMCoffsetComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>forbidOffsetEdit(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>plotDistanceDiagramm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputOkButton()</slot>
  <slot>showFollowerRadius(QString)</slot>
  <slot>forbidOffsetEdit(QString)</slot>
  <slot>plotDistanceDiagramm()</slot>
 </slots>
</ui>
